gulpfile.js/tasks/changelog.js

```diff
@@ -22,56 +22,41 @@ var runCommand = function (cmd) {
   })
 }
 
-var getCurrentCommit = function (commit) {
-  return new Promise(function (resolve, reject) {
-    if (commit) {
-      resolve(commit)
-    } else {
-      var cmd = 'git rev-parse HEAD'
-      runCommand(cmd).then(resolve)
-    }
-  })
-}
-
-var getChangedFiles = function (commit) {
-  if (!commit) {
-    throw new Error('No commit given')
+function getFileChanges(path) {
+  if (!path) {
+    throw new Error('No path given')
   }
 
-  var ref = process.env.GIT_PREVIOUS_SUCCESSFUL_COMMIT || 'master'
+  let cmd = `git diff master -- ${path}`
```

Comment:
Nice! 👍

Reply:
Actually, isn't this going to fail the build when run on master (i.e. on Jenkins)?
The `process.env.GIT_PREVIOUS_SUCCESSFUL_COMMIT` is a Jenkins env var to ensure it still checks between builds.